src/vs/editor/common/controller/oneCursor.ts

```diff
@@ -622,15 +632,55 @@ export class OneCursorOp {
 			validatedViewPosition = cursor.convertModelPositionToViewPosition(validatedPosition.lineNumber, validatedPosition.column);
 		}
 
+		return this.move(cursor, inSelectionMode, validatedViewPosition, eventSource, ctx);
+	}
+
+	public static move(cursor:OneCursor, inSelectionMode: boolean, to:editorCommon.IPosition | string, eventSource: string, ctx: IOneCursorOperationContext): boolean {
```

Comment:
@alexandrudima Old one is called **moveTo** and new one is called **move**. May be this differentiation is ok.